webpack/react_app/components/TargetingHosts/TargetingHostsHelpers.js

```diff
@@ -0,0 +1,9 @@
+import { getURI } from 'foremanReact/common/urlHelpers';
+
+export const getApiUrl = (searchQuery, pagination) =>
+  getURI()
+    .search('')
+    .addQuery('search', searchQuery)
```

Comment:
Empty search eg :`/job_invocations/15?search=&page=1&per_page=20&format=json` causes the hosts list to be empty for queued jobs for me, can you not add the search if its empty or maybe fix it in the controller?